eng/pipelines/official.yml

```diff
@@ -26,7 +26,7 @@ parameters:
 - name: RunMonoTestsOnMac
   displayName: Run Mono tests on Mac
   type: boolean
-  default: false
+  default: true
```

Comment:
I'm not sure there's much benefit to running any tests on official builds. When they hang causing a timeout, the entire CI job fails, causing problems when we need to insert. Even if they don't fail, when we have to service an already released branch, it adds delays since we need to wait for slow tests to finish before we can start the insertion.

Reply:
Agreed, originally I just reverted the commit that disabled them but I'll push a commit that leaves them off for official builds so they only run on PrivateDev